.github/workflows/ci-docs.yaml

```diff
@@ -4,7 +4,7 @@ name: ci-docs
 # Trigger the workflow when:
 on:
   # A push occurs to one of the matched branches.
-  push:
+  pull_request:
```

Comment:
This doesn't seem right, was this for debugging?

Reply:
Yes, this is just for debugging.